.rubocop_local.yml

```diff
@@ -1,2 +1,5 @@
 # GlobalVars:
 #   AllowedVariables:
+AllCops:
+  Exclude:
```

Comment:
I think we should exclude all Rails conventions here.  That's what's causing the things like the rake environment stuff.